### PR TITLE
Fix/dgj 169 auto populate data

### PR DESCRIPTION
**Pull request by iman-jamali-fw**
## Summary

This pr fixes the issue with auto-populate data from ODS reported in [this ticket](https://app.zenhub.com/workspaces/digital-journeys-61c10fe84c8bcb001491c5f5/issues/bcgov/digital-journeys/169)

## Changes
- A small fix to `employeeDataService.py`.
- The flag that previously was added to stop setting the default value s(from ODS) to the form was removed.
- Auth token refresh was no longer dispatches the token. This dispatch was the root cause that made the form refresh every 5 mins (depending on the auth token refresh interval value).

## Notes
Please read more about the auth token refresh issue [here ](https://github.com/bcgov/digital-journeys/pull/149)
We may want to watch in case any issue came up after stopping auth token dispatch.

### CODE REVIEW

File: forms-flow-web/src/services/UserService.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        console.warn("not authenticated!");
+        doLogin();
+      }
+    });
+};
+let refreshInterval;
+const refreshToken = (store) => {
+  refreshInterval = setInterval(() => {
+    KeycloakData && KeycloakData.updateToken(-1).then((refreshed)=> {
+      if (refreshed) {
```
Comment (iman-jamali-fw) [inline forms-flow-web/src/services/UserService.js:77]: Only this part was changed

File: forms-flow-web/src/services/UserService.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      } else {
+        console.warn("not authenticated!");
+        doLogin();
+      }
+    });
+};
+let refreshInterval;
+const refreshToken = (store) => {
+  refreshInterval = setInterval(() => {
+    KeycloakData && KeycloakData.updateToken(-1).then((refreshed)=> {
```
Comment (simensma-fresh) [inline forms-flow-web/src/services/UserService.js:76]: If I remember correctly, this one was `updateToken(5)` instead of `-1` 🙃

Comment (simensma-fresh) [reply forms-flow-web/src/services/UserService.js:76]: Just so we refresh the token when it has < 5 min left until expiry

**Review by simensma-fresh — CHANGES_REQUESTED** [review]

File: forms-flow-web/src/services/UserService.js (hunk 2 above)
Comment (iman-jamali-fw) [reply forms-flow-web/src/services/UserService.js:76]: Oops! You're right. let me fix it.

**Review by simensma-fresh — APPROVED** [review]

**Pull request merged**